server/storage/mvcc/index.go

```diff
@@ -22,14 +22,14 @@ import (
 )
 
 type index interface {
```

Comment:
This interface should be `Revision` based. Requests are about a specific revision, not key in bucket.